### PR TITLE
Milestone 2 - Calculator App

### PR DESCRIPTION
**Pull request by Azeem838**
Hi

Please find my code below for Milestone 2 of the Calculator App project.
The following components were added: 
1. App
2. Button
3. ButtonPanel
4. Display

Thank you

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import ButtonPanel from './ButtonPanel';
+import Button from './Button';
+
+export default function App() {
+  return (
+    <div id="buttonLayout">
+      <Button />
+      <ButtonPanel />
+    </div>
+  );
```
Comment (jofer86) [inline src/components/App.js]: - As stated here 
image
We should be importing button panel and display. Please do make the adjustments.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <button type="button" id="decimal">
+          .
+        </button>
+        <button type="button" id="equal">
+          equal
+        </button>
+      </div>
+    </div>
+  );
+}
```
Comment (jofer86) [inline src/components/ButtonPanel.js]: - For this file we should be using the button component, as in the React component, currently, I can see only the button HTML tag present and not the component.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## `const STATUS = "CHANGES REQUIRED"`⚠️ ⚠️ ⚠️ 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- Below you will find the comments in your code, please do tend to them and as for a re-review.
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Review by bellom — APPROVED** [review]
## Hi @Azeem838  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**